x-pack/plugins/security_solution/public/flyout/document_details/right/components/graph_preview.tsx

```diff
@@ -77,7 +77,13 @@ export const GraphPreview: React.FC<GraphPreviewProps> = memo(
         defaultMessage="An error is preventing this alert from being visualized."
       />
     ) : (
-      <React.Suspense fallback={<LoadingComponent />}>
+      <React.Suspense
```

Comment:
after the graph loads we remove the padding around it, however since we lazy load the component I found that it needs to be added here :\